### PR TITLE
[686] Add end date to 686 divorce

### PR DESCRIPTION
**Pull request by digitaldrk**
## Description of change
We are in a testing phase with our friends at BGS. They informed us we are not sending date of divorce and that we need to send it as an `end_date`. This PR does that.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#12425

### CODE REVIEW

File: lib/bgs/dependents.rb [hunk 1]
```diff
@@ -88,6 +88,7 @@ def report_divorce
           divorce_state: formatted_info['divorce_state'],
           divorce_city: formatted_info['divorce_city'],
           divorce_country: formatted_info['divorce_country'],
+          end_date: formatted_info['end_date'],
```
Comment (erluti) [inline lib/bgs/dependents.rb:91]: should the expected hash in the spec include an end_date now?

Comment (digitaldrk) [reply lib/bgs/dependents.rb:91]: Yep, I'm glad you mentioned that bc it made me realize we needed another attribute. Added 👍

**Review by kathleencrawford — APPROVED** [review]
LGTM 👍

**Review by erluti — APPROVED** [review]
LGTM

**Pull request merged**